pyodide_build/pypabuild.py

```diff
@@ -254,20 +285,14 @@ def make_command_wrapper_symlinks(symlink_dir: Path) -> dict[str, str]:
 
 
 @contextmanager
-def _create_symlink_dir(env: dict[str, str], build_dir: Path | None):
-    if build_dir:
-        # If we're running under build-recipes, leave the symlinks in
-        # the build directory. This helps with reproducing.
-        symlink_dir = build_dir / "pywasmcross_symlinks"
-        shutil.rmtree(symlink_dir, ignore_errors=True)
-        symlink_dir.mkdir()
-        yield symlink_dir
-        return
-
-    # Running from "pyodide build". Put symlinks in a temporary directory.
-    # TODO: Add a debug option to save the symlinks.
-    with TemporaryDirectory() as symlink_dir_str:
-        yield Path(symlink_dir_str)
+def _create_symlink_dir(
+    env: dict[str, str], build_dir: Path, no_isolation: bool = False
```

Comment:
no_isolation parameter is not used anymore

Reply:
Good catch – dropped in 920aec7a56a778df4bf25961ebff9056f1655803.